api/api-webhook-v1/src/main/java/com/thoughtworks/go/apiv1/webhook/request/mixins/bitbucketcloud/BitBucketCloudAuth.java

```diff
@@ -63,7 +63,7 @@ private String getTokenFromHeader(Request request) {
             }
 
             String encodedCredentials = new String(Base64.getDecoder().decode(credentials), UTF_8);
-            return encodedCredentials.contains(":") ? split(encodedCredentials, ":", 2)[1] : encodedCredentials;
+            return encodedCredentials.contains(":") ? split(encodedCredentials, ":", 2)[0] : encodedCredentials;
```

Comment:
Wouldn't this take the username and not the personal token?

Reply:
In our API docs, we have mentioned to put the webhook secret as username and some random value as password: https://api.gocd.org/current/#hosting-on-bitbucket
I think Bitbucket does not recognize personal token in url - it only takes `username:password` format.

Reply:
@kritika-singh3 ah ok, disregard my first point in my review summary then